apps/web/components/templates/challenges/all/allChallenges.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChallengeItem } from 'molecules/challengeItem/challengeItem';
+
+type challenge = {
+	title: string;
+	description: string;
+	designs: string;
+	difficulty: string;
+	image: string;
```

Comment:
Is `difficulty` really a string? Maybe it would be better to narrow this type a little bit to a union of strings e.g. `'advanced' | 'medium' | 'easy'`. What do you think about it? 😅

Reply:
Types will be based on data from BE, so for now there is a string.